Billing.API/Startup.cs

```diff
@@ -35,10 +35,10 @@ public void ConfigureServices(IServiceCollection services)
 
         public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
         {
-            if (env.IsDevelopment())
-            {
-                app.UseDeveloperExceptionPage();
-            }
+            //if (env.IsDevelopment())
```

Comment:
What do you think about explain why we have commented code?

```suggestion
            // Removed the condition IsDevelopment so the errors are thrown directly to the client 
            // and it is easy to debug, consider restore the old behavior in the future.
            //if (env.IsDevelopment())
```

Reply:
sure